lib/openstudio-standards/prototypes/common/objects/Prototype.Model.rb

```diff
@@ -2692,6 +2692,42 @@ def model_set_building_north_axis(model, north_axis)
     return true
   end
 
+  # Calculate a model's window or WWR
+  # Disregard space conditioning (assume all spaces are conditioned)
+  # which is true for most of not all prototypes
+  #
+  # @param model [OpenStudio::Model::Model] OpenStudio model object
+  # @param wwr [Boolean]
+  # @return [Numeric] Returns window to wall ratio (percentage) or window area.
+  def model_get_window_area_info(model, wwr = true)
```

Comment:
I'm fine with this, however, I wonder if this shouldn't be a wrapper for the `model_get_window_area_info_for_orientation` method we have in the `AppendixG_Dev` branch. It could have avoid redundant code, although this is probably faster. I'm fine either way, just wanted to point it out.